pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -197,9 +200,151 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 		}
 	}
 
+	//Begin AMQ resource reconcile
```

Comment:
```suggestion
	//#region AMQ resource reconcile
```

This, in conjunction with the related `//#endregion` after this block, will allow certain IDEs to visually fold this block.